src/list.tsx

```diff
@@ -475,14 +478,14 @@ const RowContent = React.forwardRef<TouchableElement, RowContentProps>((props, r
                     [styles.pointer]: !disabled,
                 })}
                 {...interactiveProps}
-                role={role}
+                role={touchableRole ?? role}
```

Comment:
I'd say this is a bug.
If you override `role` but not `touchableRole`, we shouldn't set both roles to the same value.
I know you did this to preserve old behavior, but I'd consider old behavior a bug

Reply:
So lets do a major version for this bug to be fixed, no ?

Reply:
I think this change doesn't deserve a major release.
I see this more as a bugfix than as breaking change. The possible breakage cases in product code are quite small and easy to fix if any. The alternative to preserve the old behavior would be generating invalid output when role is overriden but touchableRole isn't, I don't like that